packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -365,6 +420,22 @@ export class ProfilesCache {
         this.allProfiles.push(...allProfiles);
     }
 
+    // check correct merging of a single profile
+    protected async checkMergingConfigSingleProfile(
+        profile: imperative.IProfileLoaded
+    ): Promise<imperative.IProfileLoaded> {
+        const baseProfile = await this.fetchBaseProfile();
+        if (
+            (baseProfile?.profile.host !== profile?.profile.host ||
+                baseProfile?.profile?.port !== profile?.profile.port) &&
```

Comment:
I believe there are open issues about this exact lines of code.
I don't think it needs to be resolved here since it already cover the most common cases.

But we may need to loosen up these to support special cases (i.e. host and port in different locations)

Reply:
I can open a quick PR for the deprecated and this issue, I thought I had added the `(baseProfile?.profile?.host && baseProfile?.profile?.port) && ...` to this check, I remember testing it and it helped with situations where the base was only holding certain info